README.md

```diff
@@ -0,0 +1,33 @@
+ansible-role-pytest-rpc
+=======================
+
+This role will prepare RPC-O deployments for testing via Molecule. This
+role is meant to be used in conjunction with the "[pytest-rpc](https://github.com/rcbops/pytest-rpc)"
+plug-in.
+
+Requirements
+------------
+
+This role requires an existing rpc-openstack deployment and a JSON file
+containing the ansible dynamic inventory for that infrastructure. This role is
+assumed to be executed from the "Deployment Host" for said OpenStack
+deployment.
+
+How to Use Role
+---------------
+
+Top level role:
+
+    - hosts: hosts
+      roles:
+        - role: ansible-role-pytest-rpc
+        
+Including the role in the "main.yml" tasks file:
+
+    - import_role:
+        name: ansible-role-pytest-rpc
+
+License
+-------
+
+Apache 2.0
```

Comment:
You have indicated that this is licensed under the apache 2.0 license, but the PR does not include the appropriate license file.

https://www.apache.org/licenses/LICENSE-2.0

Ref: https://help.github.com/articles/licensing-a-repository/

Reply:
@johnduarte [The Apace License file is already present in the initial commit for the repo.](https://github.com/rcbops/ansible-role-pytest-rpc/blob/master/LICENSE)